test/images/netexec/netexec.go

```diff
@@ -357,6 +357,10 @@ func startUDPServer(udpPort int) {
 	serverAddress, err := net.ResolveUDPAddr("udp", fmt.Sprintf(":%d", udpPort))
 	assertNoError(err)
 	serverConn, err := net.ListenUDP("udp", serverAddress)
+	if err != nil {
+		assertNoError(err)
+		return
```

Comment:
`return` is not needed here, which will not be run.

Reply:
Oh, I didn't notice log.Fatal(), thank you for the review.

Reply:
Seems ```assertNoError(err)```  is just enough?

Reply:
sure